src/app/shared/components/List/index.tsx

```diff
@@ -178,7 +178,7 @@ ListProps<T, F>): JSX.Element {
     'ctrl+a, cmd+a',
     (e) => {
       e.preventDefault();
-      handleKeyPress(selectAllItems);
+      handleKeyPress(selectAllItems());
```

Comment:
This is not correct, it works because you are executing `selectAllItems` directly, to execute correctly `handleKeyPress` you must call `handleKeyPress` returned function.
Change `handleKeyPress(selectAllItems())`; to `handleKeyPress(selectAllItems)();`

Reply:
It works that way, thank you for telling me the correct way to use that function.